torch/_inductor/codegen/triton.py

```diff
@@ -726,65 +733,56 @@ def indexing(
         index = self.simplify_indexing(index)
         index_vars = index.free_symbols
         index_str = texpr(self.rename_indexing(self.codegen_indexing(index)))
-        indirect_indexing = self.is_indirect_indexing(index)
+
+        mask_vars: Set[str] = set()
+        for var in index_vars:
+            if var.name.startswith("tmp"):
+                # indirect indexing
+                cse_var = self.cse.varname_map[var.name]
```

Comment:
You can check if `varname_map` has the necessary key before this call, for a better error message.